src/lib/aggregations/get-date-histogram.ts

```diff
@@ -2,6 +2,13 @@ import { Document, Key } from 'lib/common'
 import { ExtractEnumValues } from 'lib/utils'
 import { CalendarIntervalName, CalendarIntervalQuantity } from 'lib/enums'
 
+export type DateHistogramAggregationOptions = {
+    /**
+     * @description By default all buckets between the first bucket that matches documents and the last one are returned.
+     */
+    min_doc_count?: number // eslint-disable-line camelcase
```

Comment:
should we convert the `snake case` to `camel case`?

Reply:
afaik we are currently only using the `snake case` - we should leave it for now @ada-krupa and later we can figure out how to create types and support for both cases 🙏 (or only one) cc @WadasGrzegorz @adrianboratyn